site/content/blog/2023-08-25-community-tips-and-tricks/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Community - Tips and Tricks"
+summary: >
+  News about a community area to contribute ZAP usage tips and tricks.
+images:
+- https://www.zaproxy.org/blog/2023-08-25-community-tips-and-tricks/images/zapbot.TandT.gif
+addSocialPreview: true
+type: post
+date: "2023-08-25"
+authors:
+    - thorin
+---
+
+![](images/zapbot.TandT.gif)
+
+# Community - Tips and Tricks
+
+We've established a [community area](https://github.com/zaproxy/community-scripts/tree/main/other/tips) to publish people's Tips and Tricks for using ZAP and its add-ons. Please feel free to open PRs with your best ideas.
```

Comment:
I like it but it looks very sparce at the moment :/
Are there any other tips-n-tricks we could include?
I dont have huge expectations for the community to share loads more with us ..

Reply:
I don't have anything else ready off the top of my head. But it has to start somewhere.